Shoko.Plugin.Abstractions/Extensions/LanguageExtensions.cs

```diff
@@ -132,31 +146,48 @@ public static TitleLanguage GetTitleLanguage(this string lang)
             "TK" or "TUK" => TitleLanguage.Turkmen,
             "UG" or "UIG" => TitleLanguage.Uighur,
             "UZ" or "UZB" => TitleLanguage.Uzbek,
-            "CY" or "CYM" => TitleLanguage.Welsh,
+            "CY" or "CYM" or "WEL" => TitleLanguage.Welsh,
             "XH" or "XHO" => TitleLanguage.Xhosa,
             "YI" or "YID" => TitleLanguage.Yiddish,
             "YO" or "YOR" => TitleLanguage.Yoruba,
             "ZU" or "ZUL" => TitleLanguage.Zulu,
             "UR" or "URD" => TitleLanguage.Urdu,
+
+            "X-JAT" => TitleLanguage.Romaji,
+            "X-ZHT" => TitleLanguage.Pinyin,
+            "X-KOT" => TitleLanguage.KoreanTranscription,
+            "X-THT" => TitleLanguage.ThaiTranscription,
+
+            #region Backwards Compat
+            
+            "CZ" => TitleLanguage.Czech,
+            "ML" => TitleLanguage.Dutch,
+            "GR" => TitleLanguage.Greek,
+            "SE" => TitleLanguage.Swedish,
+            "UA" => TitleLanguage.Ukrainian,
+            "BD" => TitleLanguage.Bengali,
+            "X-THA" => TitleLanguage.ThaiTranscription,
             "GREEK (ANCIENT)" => TitleLanguage.Greek,
             "JAVANESE" or "MALAY" or "INDONESIAN" => TitleLanguage.Malaysian,
             "PORTUGUESE (BRAZILIAN)" => TitleLanguage.BrazilianPortuguese,
             "THAI (TRANSCRIPTION)" => TitleLanguage.ThaiTranscription,
             "CHINESE (SIMPLIFIED)" => TitleLanguage.ChineseSimplified,
             "CHINESE (TRADITIONAL)" => TitleLanguage.ChineseTraditional,
             "CHINESE (CANTONESE)" or "CHINESE (MANDARIN)" or
-            "CHINESE (UNSPECIFIED)" or "TAIWANESE" => TitleLanguage.Chinese,
+                "CHINESE (UNSPECIFIED)" or "TAIWANESE" => TitleLanguage.Chinese,
```

Comment:
extra whitespace (or so gh is telling me)

Reply:
The space was added because it's continuing the expression from the previous line